File: code-coach/free/easy/hovercraft.java
import java.util.Scanner;

public class Program {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int sales = Integer.parseInt(scanner.nextLine());
        int invest = 10 * 2000_000 + 1000_000;
        long monthly = (long) sales * 3000_000;

        if (monthly < invest)
            System.out.println("Loss");
        else if (monthly == invest)
            System.out.println("Broke Even");
        else
            System.out.println("Profit");

        scanner.close();
    }
}

File: code-coach/free/easy/hovercraft.c
#include <stdio.h>

int main() {
    int sales;
    scanf("%d", &sales);

    long long invest = 10LL * 2000000 + 1000000;
    long long monthly = (long long)sales * 3000000;

    if (monthly < invest) {
        printf("Loss\n");
    }
    else if (monthly == invest) {
        printf("Broke Even\n");
    }
    else {
        printf("Profit\n");
    }

    return 0;
}